### DISCUSSION TITLE
feature request remember last session

### DISCUSSION THREAD

**Discussion by simon1tan** [doublecmd/doublecmd#1145] [Suggestions]
It would be appreciated that when this program closes, the next time it opens, it restores where it closed.  Currently, upon opening, all tabs are lost, paths default to /usr/lib/doublecmd/. Also, I would suggest always default to user's home.

**Comment by alexx2000** [comment]
This is not entirely true. Currently  all open tabs are saved, except for tabs with non-local directories. If you have problem with it then check write access to doublecmd.xml.
